docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -464,7 +506,7 @@ We configured Kafka itself to:
 * Add the TLS certificates of the Otterize credentials operator.
 * Set the default ACL for all topics to allow anonymous access.
 
-### Per-client setups:
+### Per-client setups
```

Comment:
Can't leave a comment on the right line, but I meant to comment on the 'Bonus tutorial' below: I guess this is why client-other is there. I suggest we still remove it to simplify.